server/src/store/Memory.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Memory store more for testing purpose than production use.
+ */
+export class MemoryStore {
+  constructor() {
+    this.store = {};
+    this.hasSocketAdapter = false;
+  }
+
+  async get(key, field) {
+    if (this.store[key] === undefined || this.store[key][field] === undefined) {
+      return '';
```

Comment:
For consistency with Redis, this should probably [return null](https://redis.io/commands/hget) if the field doesn't exist.
```suggestion
      return null;
```